### ISSUE TITLE
Bash `file` tool

### ISSUE THREAD

**Issue by davis68** [swcarpentry/shell-novice#1177]
(`file` is hard to parse as a name, so apologies if this has been discussed previously.)

The `file` tool  can be used to identify and analyze file types in detail.  It's not fooled by file suffixes and it can provide more detail about structure and compression than one can easily determine through GUI or other means.

This has been useful enough for me that I put it forward for consideration as an inclusion in the `shell-novice` lesson.  Some form of `file` is universal to Unix-like systems.

If the consensus is that it would be profitable to have an optional exercise available using this tool and the current data set, I will write one up or work with a instructor candidate to compose one.

- [`file` man page](https://linux.die.net/man/1/file)
- [A blog post about `file`](https://pcarleton.com/2016/09/10/mystery-file-format/)

**Comment by gdevenyi** [comment]
Hi. Thanks for the suggestion, but I think this is out of scope for shell-novice. This seems ideal for https://github.com/carpentries-incubator/shell-extras/ and we would welcome a contribution there.
